crates/uv/tests/it/pip_freeze.rs

```diff
@@ -574,3 +574,46 @@ fn freeze_prefix() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn freeze_exclude() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let requirements_txt = context.temp_dir.child("requirements.txt");
+    requirements_txt.write_str("MarkupSafe==2.1.3\ntomli==2.0.1")?;
+
+    let prefix = context.temp_dir.child("prefix");
+
+    // Install packages to a prefix directory.
+    context
+        .pip_install()
+        .arg("-r")
+        .arg("requirements.txt")
```

Comment:
I think you should drop the requirements.txt entirely and just use `MarkupSafe` and `ntomli` as arguments here.